content/release-10-18-0/datahub-10-18-0-bundle/10_18_0.md

```diff
@@ -0,0 +1,45 @@
+---
+weight: 40
+title: Release 10.18.0
+layout: bundle
+---
+
+Cumulocity IoT DataHub Release 10.18 includes the following improvements, limitations, and known issues:
+
+### New version of internal query engine
+
+Cumulocity IoT DataHub now uses version 24 of Dremio as query engine. The new version comes along with various features and improvements like a revamped user interface, simplified interaction with PowerBI, improved query performance.
+
+### Efficient ODBC connectivity
+
+The ODBC driver required for running queries against Dremio now relies on Apache Arrow, whose columnar memory format allows for a more efficient ODBC-based query execution.
```

Comment:
Would add that the legavy driver continues to work and that the new driver will use a different port (32010 by default)